app/services/flux/merge_js_and_ecma.rb

```diff
@@ -0,0 +1,114 @@
+module Flux
+  class MergeJSAndECMA
+    include Mandate
+
+    JS_SHA = "6a8a5a41b89a45008b46ca18ff7ea800baca1c4c"
+
+    def call
+      lock_js_shas
+      repoint_solutions
+      rename_js_exercises
+      update_tracks
+      creates_track_mentorships
+      fix_unlocking
+    end
+
+    private
+
+    def lock_js_shas
+      Solution.where(exercise_id: js.exercises).update_all(git_sha: JS_SHA)
+    end
+
+    def repoint_solutions
+      # Delete any unstarted ecma exercises
```

Comment:
These will get readded by the FixUnlocking script, but doing this will speed things up a lot.